client/features/accountPage/Account.jsx

```diff
@@ -1,33 +1,70 @@
 import React from 'react';
+import axios from 'axios';
 import './Account.css';
 
-const Account = props => (
-  <div className="accountPage">
-    <div class="users-header">
-      <h1 class="users-title">Streamfinder</h1>
-      <input class="users-searchBar" type="text" placeholder="Search For Movies" />
-      {/* <img class="users-homeIcon" src="../home/homeIcon.png" />
-      <img class="users-userIcon" src="../home/userIcon.png" /> */}
-    </div>
-    <hr />
-    <h1 class="users-account">Account</h1>
-    <div class="movieModule">
-    </div>
-    <hr />
-    <div class="memberInfo">
-      <h2>Member Info</h2>
-      <ul>Username: username</ul>
-      <ul>Email: email</ul>
-      <ul>Password: password</ul>
-    </div>
-    <hr />
-    <div class="subscriptionInfo">
-      <h2>Subscription Info</h2>
-      <ul>Subscriptions: </ul>
-      <ul>Total Cost: </ul>
-      <ul>Most used subscription: </ul>
-    </div>
-  </div>
-);
+class Account extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: 'userjkhjkhName',
+      email: 'emajnknil',
+      password: 'paskjnjknsword',
+      platforms: [
+        {name: 'Netflix', id: 'netflix', isSelected: true, cost: 17.99},
+        {name: 'Amazon', id: 'amazon', isSelected: false, cost: 16.99},
+        {name: 'HBO', id: 'hbo', isSelected: true, cost: 15.99},
+        {name: 'Disney', id: 'disney', isSelected: false, cost: 17.99}
+      ]
+    }
+  }
+
+  componentDidMount() {
+    axios.get('/auth/user', this.state)
```

Comment:
Have you considered making `/users` its own route? That way the Home feature could store their watch history, reviews, etc. on their account.